Server/VikingVault.Services/AccountService.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Text;
+using VikingVault.API.Controllers;
+using VikingVault.DataAccess;
+using VikingVault.Services.Abstractions;
+
+namespace VikingVault.Services
+{
+    public class AccountService:IAccountService
+    {
+        private readonly VikingVaultDbContext _dbContext;
+        private IList<UserAccount> accountsList;
+
+        public AccountService(VikingVaultDbContext dbContext)
+        {
+            _dbContext = dbContext;
+            accountsList = new List<UserAccount>();
+        }
+
+        public UserAccount FindById()
+        {
+            return null;
```

Comment:
Is this not implemented yet? If it's not it's better to return a new exception of type `NotImplementedException`

Reply:
It is okay, I talked to Vasi about it, he agreed to return null there, that result is needed in the controller.